flutter_vlc_player/lib/src/vlc_player_controller.dart

```diff
@@ -29,6 +31,10 @@ typedef RendererCallback = void Function(VlcRendererEventType, String, String);
 ///
 /// After [dispose] all further calls are ignored.
 class VlcPlayerController extends ValueNotifier<VlcPlayerValue> {
+  VlcPlayerPlatform getVlcPlayerPlatform() {
+    return _vlcPlayerPlatform;
+  }
```

Comment:
We don't need to expose this as a public method.

We can put the platfrom global variable (`_vlcPlayerPlatform`) into a separate file and make it public, and then just don't export it in `lib/vlc_flutter_player.dart`.

Reply:
fixed